dir.props

```diff
@@ -276,7 +276,8 @@
     <NETCoreAppTestSharedFrameworkPath>$(NETCoreAppTestRootPath)shared/Microsoft.NETCore.App/$(NETCoreAppTestSharedFxVersion)/</NETCoreAppTestSharedFrameworkPath>
 
     <!-- Constructed shared fx path for testing -->
-    <TestSharedFxDir>$(NETCoreAppTestRootPath)</TestSharedFxDir>
+    <TestSharedFxDir Condition="'$(_bc_TargetGroup)' != 'netfx'">$(NETCoreAppTestRootPath)</TestSharedFxDir>
+    <TestSharedFxDir Condition="'$(_bc_TargetGroup)' == 'netfx'">$(RuntimePath)</TestSharedFxDir>
```

Comment:
I wouldn't bother setting it for netfx I would just add a Condition for BinplaceTestSharedFramework=true as it only applies in that case.